rj_gameplay/stp/utils/world_state_converter.py

```diff
@@ -179,6 +182,7 @@ def build_game_info(
                 play_state_msg.placement_point.y,
             ]
         ),
+        np.array([our_team_msg.score, their_team_msg.score]),
```

Comment:
this can just be a normal list (change the typing in stp/rc.py to match, too)
```suggestion
        [our_team_msg.score, their_team_msg.score],
```